misc/qutebrowser.appdata.xml

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Copyright 2017 suve <veg@svgames.pl> -->
+<component type="desktop">
+	<id>qutebrowser.desktop</id>
+	<metadata_license>CC-BY-SA-3.0</metadata_license>
+	<project_license>GPL-3.0</project_license>
+	<name>qutebrowser</name>
+	<summary>A keyboard-driven web browser</summary>
+	<description>
+		<p>
+			qutebrowser is a keyboard-focused browser with a minimal GUI.
+			It was inspired by other browsers/addons like dwb and Vimperator/Pentadactyl, 
+			and is based on Python and PyQt5.
+		</p>	
+	</description>
+	<launchable type="desktop-id">qutebrowser.desktop</launchable>
```

Comment:
Maybe add:

- [categories tag](https://www.freedesktop.org/software/appstream/docs/chap-Metadata.html#tag-categories)
- [provides tag](https://www.freedesktop.org/software/appstream/docs/chap-Metadata.html#tag-provides) for a `qutebrowser` binary
- [mimetypes tag](https://www.freedesktop.org/software/appstream/docs/chap-Metadata.html#tag-mimetypes)

Not sure what they're all used for, though.

Reply:
I don't think mimetypes are necessary, since the docs say "For applications, the metadata will automatically be fetched from their .desktop files by the distribution's metadata generator"